roles/rhscl-postgresql-apb/defaults/main.yml

```diff
@@ -2,5 +2,6 @@
 postgresql_user: "admin"
 postgresql_volume_size: "1Gi"
 postgresql_version: "9.6"
+postgresql_admin_password: "{{ lookup('password', '/tmp/admin_password length=20 chars=ascii_letters,digits') }}"
```

Comment:
I remember @jmontleon having required passwords in order to support updates later. Not sure if that applies here, but figure I should call attention to it.